### PR TITLE
removed GQL libraries, using fetch

### PR DESCRIPTION
**Pull request by dkisonas**

### CODE REVIEW

File: src/app/page.jsx [hunk 1]
```diff
@@ -16,23 +15,44 @@ async function getProjects() {
...
+`;
+
+  const data = await fetchGraphQL(query);
+  return data;
 }
 
 export default async function Home() {
-  const projects = await getProjects();
+  const { heroes } = await getProjects();
+  const { posts } = await getPosts();
```
Comment (mbitc) [inline src/app/page.jsx:49]: maybe initiate query by name and use only fetchGraphQL function to get const { heroes } and const { posts }

**Review by mbitc — CHANGES_REQUESTED** [review]
one place to look

File: src/app/page.jsx (hunk 1 above)
Comment (dkisonas) [reply src/app/page.jsx:49]: having getProjects and getPosts makes the code easier to read, it's possible to reuse these methods for other pages, and if we ever change how we "getProjects" or "getPosts", we only have to change those functions

Comment (mbitc) [reply src/app/page.jsx:49]: ok, then done

**Pull request merged**